### PR TITLE
IPC Xenowear

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Big McThankies to Kyres McSpankies for the sprites. All of them are cosmetic except the headlights, which are basically a slightly better penlight.

### CODE REVIEW

File: html/changelogs/paradoxspace-goldenbeep.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (Alberyk) [inline html/changelogs/paradoxspace-goldenbeep.yml]: Wrong author.

**Comment by ParadoxSpace** [comment]
someone slipped non-unicode characters into my food

File: html/changelogs/paradoxspace-goldenbeep.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#################################
+
+# Your name.  
+author: ParadoxSpace
```
Comment (BurgerLUA) [inline html/changelogs/paradoxspace-goldenbeep.yml]: You should also add Kyre's name here if you're giving credit.

**Comment by BurgerLUA** [comment]
I'm curious if that "&" key will fuck up changelog formatting.

**Comment by ParadoxSpace** [comment]
let's find out :^)

**Comment by skull132** [comment]
\> More fashion

Literally. _No._

Also fix the conflicts.

**Comment by MarinaGryphon** [comment]
Fashion is at the request of the synth lore team, so it has diplomatic immunity like the Taj cloaks.
Good job, Paradox.

**Comment by skull132** [comment]
DNM until I check with the lore dev on a note. Specifically the abundance of highly gold bling.

**Comment by ParadoxSpace** [comment]
the loadout will now have the power of the golden jumpsuit stripped away

**Pull request merged**